manual.adoc

```diff
@@ -149,12 +151,21 @@ Make sure that `~/.local/bin` is listed in the `$PATH` variable and use the appr
 
 You don't have to use `~/.local/bin`, any other path like `~/.cargo/bin` or `/usr/local/bin` will work just as well.
 
+On Windows:
+
+. Download a suitable Windows specific release e.g. `rust-analyzer-x86_64-pc-windows-msvc.zip`
+. Extract the files
+. Copy the `rust-analyzer.exe` file to a suitable directory (e.g. `C:\tools`)
+. Add that directory to the user's *PATH* _environment variable_
+. Open a new `Command Prompt` or `PowerShell` to use `rust-analyzer` 
```

Comment:
There is no need to invoke the binary manually. That's your LSP capable text editors job. Running the binary on its own doesn't do anything.

Reply:
Obviously I wasn't clear here. What I meant was restart the shell to get rust-analyzer to work. Can you suggest alternative wording?